lib/mix/lib/mix/task.ex

```diff
@@ -193,7 +193,10 @@ defmodule Mix.Task do
   end
 
   def alias?(task) when is_atom(task) do
-    Mix.Project.config[:aliases][task]
```

Comment:
@npbee `x != nil` only differs from `!!x` when `x` is `false`, because `false != nil #=> true` while `!!false #=> false`, but since `[]` here returns either a non-boolean term or `nil`, they are equivalent here. Since `:aliases` is a keyword list, can't we do `Keyword.has_key?(Mix.Project.config()[:aliases], task)`?